rwengine/src/objects/PickupObject.cpp

```diff
@@ -3,6 +3,23 @@
 #include <objects/CharacterObject.hpp>
 #include <objects/PickupObject.hpp>
 
+glm::vec4 colours[14] = {
+      glm::vec4(1.f, 0.f, 0.f, 1.f) // bat, detonator, adrenaline
+    , glm::vec4(0.f, 1.f, 0.f, 1.f) // pistol
+    , glm::vec4(0.5f, 0.5f, 1.f, 1.f) // uzi
+    , glm::vec4(1.f, 1.f, 0.f, 1.f) // shotgun
+    , glm::vec4(1.f, 0.f, 1.f, 1.f) // ak47
+    , glm::vec4(0.f, 1.f, 1.f, 1.f) // m16
+    , glm::vec4(1.f, 0.5f, 0.f, 1.f) // sniper
+    , glm::vec4(0.f, 1.f, 0.5f, 1.f) // rocket
+    , glm::vec4(0.5f, 0.f, 1.f, 1.f) // flame
+    , glm::vec4(0.5f, 1.f, 0.f, 1.f) // molotov
+    , glm::vec4(1.f, 1.f, 1.f, 1.f) // grenade
+    , glm::vec4(0.5f, 1.f, 0.5f, 1.f) // bodyarmour, bribe
+    , glm::vec4(0.f, 0.f, 1.f, 1.f) // info, killfrenzy
+    , glm::vec4(1.f, 1.f, 0.f, 1.f) // health, bonus
+};
```

Comment:
Suggestion: Use original format (so a byte array or uint32 array) as it appeared in GTA memory for these structures and construct the glm vector where necessary.
That will make it easier to reuse data in other projects (which may not use glm) + it will make it easier to add new games like VC where we'd otherwise add a lot of manual work to do the conversion before adding the data here.

I'm not sure if this is how arrays are currently formated.